src/test/functional/tests/buzzPage.spec.ts

```diff
@@ -20,21 +30,68 @@ test.describe("Buzz Page", () => {
 
   test.afterEach(async ({ page }) => {
     buzzPage = new BuzzPage(page);
-    await buzzPage.deleteTheNewestPost()
+    await buzzPage.deleteTheNewestPost(false)
   });
 
   test("Post should be shared", async ({ page }) => {
+    const randomTitle = generateRandomString()
     await buzzPage.navigateToSubPage(buzzPage.buzzPageButton);
-    await buzzPage.sharePost(filePath);
+    await buzzPage.sharePost(filePath, randomTitle);
     await page.reload();
```

Comment:
Why you use page.reload() here? I think that the post should appear right away on the wall. If the expected flow is that the wall reloads itself after adding a post - then reloading it once more might cause a situation where you won't catch a bug. (Bug - post is not visible right after adding it)

Reply:
reloaded is added on purpose, and there is no auto-update, I will try to work around it but I think there is no manner,
maybe it is a bug but is not like the purpose of that repo is to practice writing a code and in order to learn it we need to have uninterrupted flow.?

Reply:
so here is the bug  Bug - post is not visible right after adding it reload is only manner to make test continue.

Reply:
This is weird, because when I test it manually on - https://opensource-demo.orangehrmlive.com/  - (not local host tho) then it reloads the page automatically. Maybe when using localhost it works differently - then OK 👌 You can keep it like this to avoid more complications.